### ISSUE TITLE
countComplete triggers over and over

### ISSUE THREAD

**Issue by llSimsll** [tomgrohl/jCountdown#19]
When my timer runs out, the countComplete keeps triggering until i reload the page. Have I done something wrong, or is this a bug?

**Comment by tomgrohl** [comment]
Without seeing your code I don't know. Please provide more information.

**Comment by llSimsll** [comment]
This is how I call the start and stop:

```
function startTimer(id, time){
    var url = 'ajax_requests.php?f=7&id=' + id + '&ti=' + time;
    var $thisTimer = $('#timer' + id);
	$.getJSON(url, function(jd) {
        if(jd.resp == 1){
            $thisTimer.attr("data-cdate", jd.ret);
            $thisTimer.on("countComplete", function() { stopTimer(id) });
            $thisTimer.countdown({dataAttr: "cdate"});
            $('#' + id).removeClass('relayOff').addClass('relayOn').button("refresh");
        }
	});
}
function stopTimer(id){
    var url = 'ajax_requests.php?f=8&id=' + id;
	$.getJSON(url, function(jd) {
        if(jd.resp == 1){
            $('#timer' + id).countdown("destroy");
            $('#' + id).removeClass('relayOn').addClass('relayOff').button("refresh");
        }
    });
}
```

**Comment by llSimsll** [comment]
Well, maybe it's not triggering all the time. But I can't run the timer again after it has run out. I have to reload the page first.

**Comment by llSimsll** [comment]
Boiling further down; this won't update the time, until I reload the page
```
$thisTimer.countdown("destroy");
$thisTimer.attr("data-cdate", jd.ret);
alert($thisTimer.attr("data-cdate")); // Alert the updated time, just for debug, works fine!
$thisTimer.countdown({dataAttr: "cdate"}); // Don't update the way it's supposed to!
```

**Comment by tomgrohl** [comment]
Ok ive managed to replicate the issue and i am correctly working on a fix.

**Comment by tomgrohl** [comment]
As a temporary fix you do the following:

```
...
$thisTimer.countdown("destroy");
$thisTimer.removeData('cdate');
...

The issue relates to the data overriding the attribute on the element. Its a strange quirk of accessing an elements attribute that is also a data value.

**Comment by llSimsll** [comment]
Okay, thanks. BTW, great plugin, keep up the good work!

**Comment by tomgrohl** [comment]
This is now fixed in 2.0.2
